Rubberduck.Refactorings/ExtractInterface/ExtractInterfaceRefactoring.cs

```diff
@@ -102,28 +122,35 @@ private void AddInterface()
 
             AddInterfaceMembersToClass(rewriter);
 
+            //We have to reparse here already, because 
```

Comment:
because what?

Reply:
Hm, I probably got distracted there. Will add the explanation.